packages/strapi-plugin-content-type-builder/admin/src/containers/ModelPage/index.js

```diff
@@ -373,7 +373,14 @@ export class ModelPage extends React.Component {
     this.setState({ attrToDelete: null, showWarning: false });
   };
 
-  handleGoBack = () => this.props.history.goBack();
+  handleGoBack = () => {
+    const {
+      location: { pathname },
+    } = this.props;
+    const backPathname = pathname.substr(0, pathname.lastIndexOf('/'));
```

Comment:
I'm not sure to understand where it should redirect the user? The homepage with the previous selected tab or just the previous view?

Reply:
It should redirect to the homepage with the tab selected at the feature where the user were. 
Ex: If he comes from a group, it redirects to the homepage with the group tab selected, even if he previously navigated in the leftmenu.